examples/community/stable_diffusion_tensorrt_txt2img.py

```diff
@@ -832,7 +835,10 @@ def __denoise_latent(
         latents = 1.0 / 0.18215 * latents
         return latents
 
+    # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline
```

Comment:
This `__decode_latent` function cannot be copied from `StableDiffusionPipeline` because they are different, [see](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L540).